suite-native/module-dev-utils/src/components/LocalFirstRelaySettings.tsx

```diff
@@ -0,0 +1,45 @@
+import { useDispatch, useSelector } from 'react-redux';
+
+import { setLocalFirstStorageRelayUrl } from '@suite-common/local-first-storage/src/labeling/labelingActions';
+import { selectLocalFirstStorageRelayUrl } from '@suite-common/local-first-storage/src/labeling/labelingSelectors';
+import { yup } from '@suite-common/validators';
+import { Button, Card, Text, VStack } from '@suite-native/atoms';
+import { Form, TextInputField, useForm } from '@suite-native/forms';
+
+export const LocalFirstRelaySettings = () => {
+    const localFirstStorageRelayUrl = useSelector(selectLocalFirstStorageRelayUrl);
+    const dispatch = useDispatch();
+
+    const form = useForm<{ localFirstStorageRelayUrl: string }>({
```

Comment:
I just checked out how's it done in native and did the same, maybe overengineered but... Standard